qt/widgets/common/src/MuonFitDataSelector.cpp

```diff
@@ -356,7 +356,23 @@ QString MuonFitDataSelector::getSimultaneousFitLabel() const {
  * @param label :: [input] Text to set as label
  */
 void MuonFitDataSelector::setSimultaneousFitLabel(const QString &label) {
-  m_ui.txtSimFitLabel->setText(label);
+  // do some checks that it is valid
+  auto safeLabel = label.toStdString();
+  auto index = safeLabel.find(".nxs");
+  if (index != std::string::npos) {
+    safeLabel = safeLabel.substr(0, index);
+  }
+  index = safeLabel.find_last_of("\\");
```

Comment:
This will only work on windows

Reply:
If I am not mistaken the load current run only works on windows, so other platforms will not see this error